### ISSUE TITLE
Reminder for counters

### ISSUE THREAD

**Issue by MF-Debug** [albertvaka/bettercounter#43]
An option to enable notification reminders or alerts for the counters on a specific time and date (hourly, daily, weekly, monthly etc.) would be fantastic.

**Comment by albertvaka** [comment]
I don't think this will ever be implemented. It's too big of a feature and it's something that can be done easily with a calendar app.

**Comment by MF-Debug** [comment]
Fair enough, I'll close this issue then.
